doc/topics/jinja/index.rst

```diff
@@ -1148,6 +1148,40 @@ Returns:
     }'
   }
 
+
+.. jinja_ref:: traverse
+
+``traverse``
+------------
+
+.. versionadded:: 2018.3.3
+
+Traverse a dict or list using a colon-delimited target string.
+The target 'foo:bar:0' will return data['foo']['bar'][0] if this value exists,
+and will otherwise return the provided default value.
+
+Example:
+
+.. code-block:: jinja
+
+  {{ {'a1': {'b1': {'c1': 'foo'}}, 'a2': 'bar'} | traverse('a1:b1', 'default') }}
+
+Returns:
+
+.. code-block:: python
+
+  {'c1': 'foo'}
```

Comment:
These code-block sections need to be a full indent for this to display properly in the docs. Can you fix this one, as well as the rest of the code-block sections?

Reply:
@rallytime these python code-block sections already exists in this `index.rst` file and renders correctly in the docs. See for example the  `compare_lists` filter doc: https://docs.saltstack.com/en/latest/topics/jinja/index.html#compare-lists

Reply:
Interesting. Usually sphinx generates a warning about that I thought. *shrug*